docs/tutorials/semantic_search/asymmetric_embedding_model.md

```diff
@@ -1,36 +1,27 @@
 # Tutorial: Running Asymmetric Semnantic Search within OpenSearch
 
-This tutorial demonstrates how generating text embeddings using an asymmetric embedding model in OpenSearch. The embeddings will be used
-to run semantic search, implemented using a Docker container. The example model used in this tutorial is the multilingual
-`intfloat/multilingual-e5-small` model from Hugging Face. 
-You will learn how to prepare the model, register it in OpenSearch, and run inference to generate embeddings.
+This tutorial demonstrates how generating text embeddings using an asymmetric embedding model in OpenSearch. The example model used in this tutorial is the multilingual
+`intfloat/multilingual-e5-small` model from Hugging Face.
+In this tutorial, you'll learn how to prepare the model, register it in OpenSearch, and run inference to generate embeddings.
 
 > **Note**: Make sure to replace all placeholders (e.g., `your_`) with your specific values.
 
 ---
 
 ## Prerequisites
 
-- Docker Desktop installed and running on your local machine.
-- Basic familiarity with Docker and OpenSearch.
+- OpenSearch installed on your machine
 - Access to the Hugging Face `intfloat/multilingual-e5-small` model (or another model of your choice).
+- Basic knowledge of Linux commands
 ---
 
-## Step 1: Spin up a Docker OpenSearch cluster
+## Step 1: Start OpenSearch locally
```

Comment:
Do we care if customer started opensearch locally or with public IP? In our blue print example, we setup OS locally.

Reply:
Ideally locally because I have a step were I make them use python to service the model which runs on localhost:8080

Reply:
if I have a OS which is running in let's say `8.8.8.8` and then if I use `8.8.8.8:8080`, won't that work?

Reply:
Not an expert but if two processes run on the same address like localhost but use different ports then it should be fine. Also the python server is just a one time occurrence just a means to get OS to download the model.

Reply:
It should work. If we setup a OS cluster in EC2 host, we can access through the public URL and the port. So I would suggest to rephrase: `Start Opensearch` and in line 22 we can say:

Run OpenSearch and ensure the following steps are completed. In this example, we set up the OS cluster locally.

Reply:
Makes sense, thank you for the suggestion (addressed in commit b11cbe1).